content/development/getting-started.html

```diff
@@ -5,9 +5,12 @@
 tags = []
 +++
 
-The information on this page is intended to explain how you can get started with Haiku development.
+<p>Developers are the driving force behind Haiku, and without them, we would not be where we are today. There's always a lot of work to be done, so this is by far one of the most important jobs anyone in the Haiku community can take up on. Consequently, it will probably require a large investment of your time. Even so, it’s hard to put into words the pleasure of seeing something you’ve built help many other people. Newcomers will need to know or be willing to learn C++, as well as the BeOS/Haiku API.
```

Comment:
I don't really agree with several parts of this:
- Developers are only one of the many tasks and not as important as you make it sound. It's possible to have as much, if not more, impact by working in other areas
- It does not have to take a significant part of your time. If you submit one 2-line patch every 10 years that's already great and totally fine. We are trying to build a low-presure environment here where everyone is welcome to contribute in accordance with their available time and level of motivation.

Reply:
Didn't want to alter the content too much while making it sound a bit more significant. Will fix.